purchase_requisition_bid_selection/model/purchase_requisition.py

```diff
@@ -440,9 +450,32 @@ def open_product_line(self, cr, uid, ids, context=None):
             del ctx['search_default_hide_cancelled']
         ctx['search_default_groupby_requisitionline'] = True
         ctx['search_default_showbids'] = True
+        ctx['search_default_selection_view'] = True
         return res
 
     @api.multi
+    def ask_validity(self):
+        ctx = self._context.copy()
```

Comment:
Why using `self._context` instead of `self.env.context` as in `update_validity`?